drivers/state.go

```diff
@@ -93,9 +93,9 @@ func (d *StateDriver) newMinerAccountActor() address.Address {
 
 	// creat a miner, owner, and its worker
 	_, minerOwnerID := d.NewAccountActor(address.SECP256K1, big_spec.NewInt(1_000_000_000))
-	_, minerWorkerID := d.NewAccountActor(address.BLS, big_spec.Zero())
+	minerWorkerPk, minerWorkerID := d.NewAccountActor(address.BLS, big_spec.Zero())
 	expectedMinerActorIDAddress := utils.NewIDAddr(d.tb, utils.IdFromAddress(minerWorkerID)+1)
-	minerActorAddrs := computeInitActorExecReturn(d.tb, builtin_spec.StoragePowerActorAddr, 0, 1, expectedMinerActorIDAddress)
+	minerActorAddrs := computeInitActorExecReturn(d.tb, minerWorkerPk, 0, 1, expectedMinerActorIDAddress)
```

Comment:
The parameter currently `1` depends on how many prior actors have been Exec'd, so I can't quite tell if that needs changing too. Should probably become a parameter to this method or a state variable..

Reply:
Yeah, probably makes sense. Though at some point we're just reimplementing the whole logic that we're trying to test